### PR TITLE
test: migrate SelectButton, SelectButtonGroup tests

### PR DESCRIPTION
**Pull request by PavelKirvelTextkernel**
Link [task](https://textkernel.atlassian.net/browse/ONEUI-395)

### CODE REVIEW

File: src/components/SelectButtonGroup/SelectButtonGroup/__tests__/SelectButtonGroup.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,240 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { SelectButtonGroup } from '../SelectButtonGroup';
+import { SelectButton } from '../../SelectButton';
+
+describe('SelectButtonGroup', () => {
+    let view;
```
Comment (andrentk) [inline src/components/SelectButtonGroup/SelectButtonGroup/__tests__/SelectButtonGroup.spec.tsx:9]: type missing

Comment (PavelKirvelTextkernel) [reply src/components/SelectButtonGroup/SelectButtonGroup/__tests__/SelectButtonGroup.spec.tsx]: :+1:

File: src/components/SelectButtonGroup/SelectButtonGroup/__tests__/SelectButtonGroup.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+                <SelectButton value="2" key="2">
+                    Option 2
+                </SelectButton>
+                <SelectButton value="3" key="3">
+                    Option 3
+                </SelectButton>
+            </SelectButtonGroup>
+        );
+
+        getButton = (number) => screen.getAllByRole('button')[number];
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectButtonGroup/SelectButtonGroup/__tests__/SelectButtonGroup.spec.tsx:47]: You mean to change it to 
image

File: src/components/SelectButtonGroup/SelectButton/__tests__/SelectButton.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    });
+
+    it('should call onChange when clicked', async () => {
+        const user = userEvent.setup();
+        await user.click(screen.getByRole('button'));
+
+        expect(onChangeMock).toHaveBeenCalledWith('button 1');
+    });
+
+    it.skip('should call onChange when Enter key was pressed on it', async () => {
```
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectButtonGroup/SelectButton/__tests__/SelectButton.spec.tsx:72]: i've fixed it

File: src/components/SelectButtonGroup/SelectButtonGroup/__tests__/SelectButtonGroup.spec.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (PavelKirvelTextkernel) [reply src/components/SelectButtonGroup/SelectButtonGroup/__tests__/SelectButtonGroup.spec.tsx]: :+1:

**Review by andrentk — CHANGES_REQUESTED** [review]
Remove Please Ignore it from the PR title.

**Review by andrentk — APPROVED** [review]

**Pull request merged**